packages/web-app-maps/src/components/LocationFolderView.vue

```diff
@@ -10,70 +10,36 @@
 </template>
 
 <script setup lang="ts">
-import { ref, unref, watch, computed, onMounted, onUnmounted } from 'vue'
-import { useLeaflet } from '../composables'
+import { ref, unref, computed, onMounted, onUnmounted } from 'vue'
+import { useLeaflet, useMapPins } from '../composables'
 import { NoContentMessage } from '@opencloud-eu/web-pkg'
 import { Resource } from '@opencloud-eu/web-client'
-import * as Leaflet from 'leaflet'
-import { LatLngExpression } from 'leaflet'
 import { useGettext } from 'vue3-gettext'
 
-// FIXME: Leaflet types seem broken?!
-const L = Leaflet as any
-
 const { resources, applicationConfig } = defineProps<{
   resources: Resource[]
   applicationConfig: Record<string, any>
 }>()
 
 const { $gettext } = useGettext()
-const { createMap, createPinIcon } = useLeaflet()
+const { createMap } = useLeaflet()
 const leafletElement = ref<HTMLElement | null>(null)
 const initialized = ref(false)
+const mapObject = ref<L.Map | null>(null)
```

Comment:
Giving it `null` initially changes the type, maybe you remember the issue we had some time ago 😬 

```suggestion
const mapObject = ref<L.Map | null>()
```

Or maybe just remove `null` from the type as well? Since it will be typed `L.Map | undefined` implicitly.

```suggestion
const mapObject = ref<L.Map>()
```